cmd/cloud_sql_proxy/cloud_sql_proxy.go

```diff
@@ -41,6 +41,7 @@ import (
 	"github.com/GoogleCloudPlatform/cloudsql-proxy/proxy/util"
 
 	"cloud.google.com/go/compute/metadata"
+	"github.com/coreos/go-systemd/v22/daemon"
```

Comment:
How big of a dependency is this? Do you know what the change in binary size is?

Reply:
Pretty small:
```shell
$ ls -l cloud_sql_proxy*
-rwxr-xr-x. 1 cape cape 16481993  5 apr 20.56 cloud_sql_proxy*
-rwxr-xr-x. 1 cape cape 16494725  5 apr 20.26 cloud_sql_proxy-with-systemd*
```